src/renderer/components/dock/terminal/terminal.ts

```diff
@@ -53,7 +54,8 @@ export class Terminal {
     return this.elem.querySelector(".xterm-viewport")!;
   }
 
-  attachTo(parentElem: HTMLElement) {
+  async attachTo(parentElem: HTMLElement) {
+    await this.dependencies.preloadFont(this.fontFamily);
```

Comment:
wouldn't it be better if we just preloaded the fonts once in a `Runnable`? That way we don't have to add these `preloadFont` calls all over the place.

Reply:
I agree. Should be easy. Example here: https://github.com/lensapp/lens/pull/6913/files#r1067800560